tests/test_places.py

```diff
@@ -67,7 +68,6 @@ def test_basic_query_street():
 
     assert resp["id"] == "35460343"
     assert resp["name"] == "9a Birnenweg"
-    assert resp["label"] == "9a Birnenweg (Label)"
```

Comment:
These tests should specify the whole response, to prevent regression in the future.

Reply:
Still needed for request about admins and addresses